view/frontend/templates/js/replacejs.phtml

```diff
@@ -540,7 +540,7 @@ $isLoadConnectJsDynamic = $block->isLoadConnectJsDynamic();
 
             close: function () {
                 popUpOpen = false;
-                trackCallbacks.onClose();
+                trackCallbacks.onClose(callbacks);
```

Comment:
Sending callbacks as parameters look a little hacky to me. Could you please explain the task maybe we can find another solution?

Reply:
Let me know if you have any input

Reply:
If we can it using a global variable on merchant side it's much better.
So we call success() and then close(). We can set some flag in success and check it in close.
Let me know if it works for your case.

Reply:
@vitaliyreznikov That is a good idea. I've re-configured and it works. Thank you
**Tracking: onSuccess**
if (data.success_url){
window.bolt_success_url = data.success_url;
}

**Tracking: onClose**
if(!window.bolt_success_url){window.location.reload();}

Reply:
Closing this PR now